pdns/ixfrdist-stats.cc

```diff
@@ -42,67 +42,64 @@ std::string ixfrdistStats::getStats() {
       stats<<"# TYPE "<<prefix<<"soa_serial gauge"<<std::endl;
     }
     if(d.second.haveZone)
-      stats<<prefix<<"soa_serial{domain="<<d.first<<"} "<<d.second.currentSOA<<std::endl;
+      stats<<prefix<<"soa_serial{domain=\""<<d.first<<"\"} "<<d.second.currentSOA<<std::endl;
     else
-      stats<<prefix<<"soa_serial{domain="<<d.first<<"} NaN"<<std::endl;
+      stats<<prefix<<"soa_serial{domain=\""<<d.first<<"\"} NaN"<<std::endl;
 
     if (!helpAdded) {
-      stats<<"# HELP "<<prefix<<"soa_checks Number of times a SOA check at the master was attempted"<<std::endl;
-      stats<<"# TYPE "<<prefix<<"soa_checks counter"<<std::endl;
+      stats<<"# HELP "<<prefix<<"soa_checks_total Number of times a SOA check at the master was attempted"<<std::endl;
+      stats<<"# TYPE "<<prefix<<"soa_checks_total counter"<<std::endl;
     }
-    stats<<prefix<<"soa_checks{domain="<<d.first<<"} "<<d.second.numSOAChecks<<std::endl;
+    stats<<prefix<<"soa_checks_total{domain=\""<<d.first<<"\"} "<<d.second.numSOAChecks<<std::endl;
     numSOAChecks += d.second.numSOAChecks;
```

Comment:
It looks like the variables from `uint64_t numSOAChecks{0}, numSOAChecksFailed{0}, numSOAinQueries{0}, numIXFRinQueries{0}, numAXFRinQueries{0}, numAXFRFailures{0}, numIXFRFailures{0};` are now unused, perhaps we could remove them? I have tested such a change in https://github.com/PowerDNS/pdns/commit/2d4713ecc13a5a58ccb6db813cf149be5df025a7

Reply:
cherry picked and also tested, thanks!